database/mssql/scripts/dbscripts/PSP_PIMS_S13_00/Test Data/04_DML_LEASE_LOAD.sql

```diff
@@ -7,93 +7,95 @@ Doug Filteau  2021-Aug-19  Initial version
 ----------------------------------------------------------------------------- */
 
 -- Insert the lease data
-INSERT INTO [dbo].[PIMS_LEASE]([LEASE_PURPOSE_TYPE_CODE], [LEASE_PURPOSE_SUBTYPE_CODE], [LEASE_STATUS_TYPE_CODE], [LEASE_PMT_FREQ_TYPE_CODE], [LEASE_PROGRAM_TYPE_CODE], [TENANT_ID], [TFA_FILE_NO], [PS_FILE_NO], [START_DATE], [RENEWAL_DATE], [EXPIRY_DATE], [LEASE_AMOUNT], [INSURANCE_START_DATE], [INSURANCE_END_DATE], [SECURITY_START_DATE], [SECURITY_END_DATE], [INSPECTION_DATE], [INSPECTION_NOTES], [LEASE_NOTES], [UNIT], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_DIRECTORY])
-  VALUES(4, 3, 'I', 'NOMINAL', 'COMMBLDG', (SELECT PERSON_ID FROM PIMS_PERSON WHERE SURNAME = 'Mouse'), 390, NULL, '19870701 00:00:00.0', NULL, '20080630 00:00:00.0', NULL, NULL, NULL, NULL, NULL, NULL, '', '', '', CURRENT_TIMESTAMP, N'Lease Seed Data', N'Lease Seed Data', CURRENT_TIMESTAMP, N'Lease Seed Data', N'Lease Seed Data')
+
+INSERT INTO [dbo].[PIMS_LEASE]([LEASE_PURPOSE_TYPE_CODE], [LEASE_RESPONSIBILITY_TYPE_CODE], [LEASE_INITIATOR_TYPE_CODE], [LEASE_LICENSE_TYPE_CODE], [LEASE_CATEGORY_TYPE_CODE], [LEASE_STATUS_TYPE_CODE], [LEASE_PMT_FREQ_TYPE_CODE], [LEASE_PROGRAM_TYPE_CODE], [LEASE_PAY_RVBL_TYPE_CODE], [TFA_FILE_NO], [PS_FILE_NO], [ORIG_START_DATE], [TERM_RENEWAL_DATE], [ORIG_EXPIRY_DATE], [LEASE_AMOUNT], [INSURANCE_START_DATE], [INSURANCE_END_DATE], [SECURITY_START_DATE], [SECURITY_END_DATE], [INSPECTION_DATE], [INSPECTION_NOTES], [LEASE_NOTES], [UNIT], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_DIRECTORY])
```

Comment:
updated this file to maintain compatibility with schema changes.

Reply:
@dfilteau please note that since I have updated these files, if you create another PSP_PIMS_SXX_XX folder, that you will need to pull in these versions of the test data otherwise you will clobber my updates.

Reply:
So, to confirm - does this test data matches the IS13 db schema or IS14 (this PR)? Please clarify

Reply:
technically supports both, although it does not populate the new subtables.